ios/Plugin/InAppBrowserPlugin.swift

```diff
@@ -53,6 +53,23 @@ public class InAppBrowserPlugin: CAPPlugin {
             return
         }
 
+    extension UIColor {
+
+     init(hexString: String) {
+     let hex = hexString.trimmingCharacters(in: CharacterSet.alphanumerics.inverted)
+        var int = UInt64()
+        Scanner(string: hex).scanHexInt64(&int)
+        let components = (
+            R: CGFloat((int >> 16) & 0xff) / 255,
+            G: CGFloat((int >> 08) & 0xff) / 255,
+            B: CGFloat((int >> 00) & 0xff) / 255
+        )
+        self.init(red: components.R, green: components.G, blue: components.B, alpha: 1)
+    }
+
+}
+}
+
```

Comment:
Again, how does gibberish input get handled? What if someone passes `!@#$%^&$#@#@#$` as the input?

Please see the output I got [here](https://github.com/Cap-go/capacitor-inappbrowser/pull/78#issuecomment-1703899861)

Reply:
I have added the error handling part now. If you use any gibberish it will revert back to white color. @WcaleNieWolny

Reply:
Okay, thanks